sdk/src/wallet/operations/transaction/options.rs

```diff
@@ -1,43 +1,65 @@
-// Copyright 2021 IOTA Stiftung
+// Copyright 2024 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
+use alloc::collections::{BTreeMap, BTreeSet};
+
 use serde::{Deserialize, Serialize};
 
 use crate::{
     client::api::input_selection::Burn,
     types::block::{
         address::Address,
         context_input::ContextInput,
-        mana::ManaAllotment,
-        output::OutputId,
+        output::{AccountId, OutputId},
         payload::{signed_transaction::TransactionCapabilities, tagged_data::TaggedDataPayload},
     },
 };
 
 /// Options for transactions
-#[derive(Debug, Clone, Serialize, Deserialize, Default)]
+#[derive(Debug, Clone, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
+#[serde(default)]
```

Comment:
Does this actually propagate to the fields? If TransactionOptions is present but one of the fields is absent in serialisation that would achieve the same?

Reply:
that's what the documentation says